src/Metadata/MetadataMatcher.php

```diff
@@ -35,12 +35,6 @@ public function withMetadataMatch(
             $fieldType = FieldType::METADATA();
         }
 
-        if ($fieldType->is(FieldType::MESSAGE_PROPERTY())
-            && ! in_array($field, ['uuid', 'createdAt', 'created_at', 'messageName', 'message_name'], true)
-        ) {
-            throw new InvalidArgumentException(sprintf('Invalid message property "%s" given', $field));
```

Comment:
I needed to remove that, because the fields have different names in pdo-event-store tables